cvat/apps/engine/static/engine/js/annotationUI.js

```diff
@@ -399,7 +399,12 @@ function setupMenu(job, task, shapeCollectionModel,
                 } finally {
                     $('#downloadAnnotationButton')[0].disabled = false;
                 }
-            }).appendTo('#downloadDropdownMenu');
+            });
+            if ((dumpSpec.display_name === 'CVAT XML 1.1 for videos' && task.mode === 'interpolation')
```

Comment:
Can we avoid code cloning somehow? It seems that in two places we have exactly the same code. Right?